apps/src/StudioApp.js

```diff
@@ -193,6 +194,18 @@ var StudioApp = function () {
    */
   this.wireframeShare = false;
 
+  /**
+   * Redux store that might be provided by the app. Initially give it an empty
+   * interface so that we can assume existence.
+   */
+  this.reduxStore_ = {
+    getState: function () {
+      return {};
+    },
+    dispatch: function () {
+    }
+  };
```

Comment:
As a matter of cleanliness, you could move the  nullimpl to redux.js as 'createFakeStore'... or heck, make a real one if you're not given one.